src/wikitextprocessor/lua/mw_wikibase.lua

```diff
@@ -23,6 +23,10 @@ function mw_wikibase.getEntityUrl(id)
    return nil
 end
 
+function mw_wikibase.getBadge(id, globalSiteId)
```

Comment:
Should be `getBadges`, other than this typos the commit LGTM.

Reply:
Oh man, yeah, the plural -s strikes again. Also the same typo in the 'test'.